docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -1,21 +1,89 @@
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 import { Alert } from '@aws-amplify/ui-react';
 
+## 2.x to 3.x
+### Installation
+Install the 3.x version of the `@aws-amplify/ui-angular` library.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-angular@3.x
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify @aws-amplify/ui-angular@3.x
+```
+
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+The following breaking changes were introduced in `@aws-amplify/ui-angular` major version `3.0`:
+
+#### Automatic signin on signup logic moved to `aws-amplify`. 
+
+##### 1. Update the handleSignUp function override call
+
+If you are overriding `Auth.signUp`, update the override function call to include the `autoSignIn` option set to enabled. If this change is not made, users will not be automatically signed in on signup.
+
+```diff
+ async handleSignUp(formData) {
+  let { username, password, attributes } = formData;
+  // custom username
+  username = username.toLowerCase();
+  attributes.email = attributes.email.toLowerCase();
+  return Auth.signUp({
+    username,
+    password,
+    attributes,
++   autoSignIn: {
++     enabled: true
++   }
+  });
+}
+
+```
+##### 2. Update to the latest version of the `aws-amplify` library.
+
+Update to the latest version of the `aws-amplify` library. You will need at least version `^4.3.29`. 
```

Comment:
Update this to 5.0, since that's the version which will be required by our major version bump

Reply:
👍